content/tools/extensions/AzureDevOpsExtension.md

```diff
@@ -1,71 +1,152 @@
 
 ## Before You Start 
 
 Before you begin to set up the extension, make sure that you have the privileges to install a Microsoft extension. If you lack the necessary privileges, you can [request the extension](https://docs.microsoft.com/en-us/azure/devops/marketplace/request-extensions?view=azure-devops-2019) for a project.
 
 ## Setup and Configuration
 
-### Step one
+Complete the following steps to configure the Contrast extension. 
+
+### Step one: Install the extension 
 
 * Follow [Microsoft's instructions](https://docs.microsoft.com/en-us/azure/devops/marketplace/install-extension?view=azure-devops-2019) to install the Contrast extension, **Contrast Integration**. 
 
-### Step two
+### Step two: Project settings
 
 * Go to your [Project Settings](https://docs.microsoft.com/en-us/azure/devops/project/navigation/go-to-service-page?view=azure-devops#open-project-settings) at the bottom of you side bar. You'll need to be part of the Project administration group or have enough permissions to alter the settings.
 
 * In the **Pipelines** section of the settings menu, select **Service connections**.
 
  <a href="assets/images/AzureDevOps_connection_settings.png" rel="lightbox" title="Service Connection Settings"><img class="thumbnail" src="assets/images/AzureDevOps_service_connection_settings.png"/></a>
 
-### Step three
+### Step three: Configure the service connection
 
 * Click over the **+ New Service connection** button (shown in the image in the previous step), and select **Contrast Server Connection**.
 * Complete all the fields with the required data. You can find the values for all the fields in the Contrast UI by going to the [**user menu > Your Account > Profile** page](user-account.html#profile).
 
 <a href="assets/images/AzureDevOps_service_connection.png" rel="lightbox" title="Service Connection fields"><img class="thumbnail" src="assets/images/AzureDevOps_service_connection.png"/></a>
 
-> **Note: ** Your **Contrast URL** should not include */Contrast* at the end; only the host is required.
+> **Note:** Your **Contrast URL** should not include */Contrast* at the end; only the host is required.
+
+## Setup and Configuration for a Task
+
+Complete the following steps to configure a task in your Azure DevOps extension. This task can be used in only an **agentless job**.
+
+### Step one: Enter Edit mode
+
+The task can be used in a release or build pipeline. Complete the appropriate steps for the pipeline you're using. 
+
+#### Release Pipeline
+
+* Enter **Edit** mode for the release pipeline you wish to add the task.
+* Select a stage for which you wish to add the task.
 
-## Configuration for Release Gate
+<a href="assets/images/AzureDevOps_release_choose_stage.png" rel="lightbox" title="Azure DevOps Release Choose Stage"><img class="thumbnail" src="assets/images/AzureDevOps_release_choose_stage.png"/></a>
 
```

Comment:
We usually don't edit the images to add red indicators. Do you have the original copies of the images that we can use instead?

Reply:
I edited them right after taking the screen shots so I do not have the originals. I will retake new ones. I reused the same images for the README and Marketplace Overview, would you like me to change those as well with images without red indicators?

Reply:
The images for the Release Gate has the red indicators to clarify where the user should be clicking which is what made me add the indicators for the task. Is that no longer the practice?

Reply:
Would you like me to replace the images with images without red indicators?

Reply:
@bryanateoan Yes, please.